frontend/src/store/eceweApp.js

```diff
@@ -184,35 +180,28 @@ export const useEceweAppStore = defineStore('eceweApp', {
             optInOrOut: this.eceweModel.fundingModel === this.fundingModelTypes[0].id ? 0 : null,
           }));
         }
+      }
+      // A payload already exists, recreate to include any new facilities which could have been added to navBarList
+      // since last creation.
+      else if (useNavBarStore().isChangeRequest) {
+        let newFac = reportChangesStore?.changeRequestMap
+          ?.get(navBarStore?.changeRequestId)
+          .changeActions?.find((el) => el.changeType == CHANGE_REQUEST_TYPES.NEW_FACILITY)?.newFacilities;
+        facilityPayload = newFac?.map((facility) => ({
+          eceweApplicationId: this.getEceweApplicationId(facility.facilityId),
+          facilityId: facility.facilityId,
+          optInOrOut: this.getOptInOrOut(facility.facilityId),
+          changeRequestId: navBarStore.changeRequestId ? navBarStore.changeRequestId : null,
+          changeRequestNewFacilityId: facility.changeRequestNewFacilityId ? facility.changeRequestNewFacilityId : null,
```

Comment:
I think ?? will work. It will either have a change request GUID or be undefined, although I don't think in this scenario it should ever be undefined. I don't think we have tried working through CR new facilities yet but I'll give it a go tomorrow and verify that